### ISSUE TITLE
Guitar Sniffer not recognizing inputs

### ISSUE THREAD

**Issue by Wafflexboy** [artman41/guitarsniffer#12]
I followed the instructions for setup, and when I launched Guitar Sniffer it never responded to inputs on my Guitar. 

I have tried uninstalling and reinstalling. I've tried using both WinPCap and NPCap, neither made a difference. 

Running Windows 10 1903, Xbox One Stratocaster guitar, Xbox one wireless adapter. All installed software was the most recent (or stable) version.

**Comment by Wafflexboy** [comment]
I also tried poking around a little with vJoy. I'm not sure where vJoy comes in or what it does but I wasn't able to get anything to change so I reset it back to default, then put in the config listed in setup here

**Comment by Dunkalunk** [comment]
So the guitar sniffer program opens and stays open?  I want to make sure you see a window like the screenshots in the instructions.

If you see the window and it's not working, make sure your guitar is working right.  Just hit the Xbox button and make sure it brings up the game bar stuff.

**Comment by Wafflexboy** [comment]
Xbox button does work, opens the game bar stuff. 

Guitar sniffer opens normally and looks almost like the first image from the setup (first image of guitar sniffer, the second showed button response). The only difference is I dont see that string of numbers/letters at the bottom.

My assumption was that the string was packet info, meaning I was either not receiving any or the software wasnt responding correctly. Windows gave no errors about not responding

**Comment by Dunkalunk** [comment]
Got it.  I always check that because if it doesn't open at all it usually has something to do with the XB1 wireless adapter or vJoy.

When you say you've tried uninstalling and reinstalling were you talking about WinPCap?  That's the other place I've had issues.  I got a new XB1 adapter and had to reinstall that for it to work again.  So I would get rid of the other NPCap and uninstall/reinstall WinPCap and see if that works.

Other than that, if you've followed the instructions exactly for ALL the programs to be installed and vJoy is set up properly I'm not sure what it would be...  MAYBE run the sniffer as administrator, that also helps some things, but if it's opening it's usually not that.  Try it anyway.

**Comment by Wafflexboy** [comment]
I've done clean installs of both WinPCap and NPCap, neither make a difference. I've tried running sniffer as admin and that doesn't seem to make a difference.

**Comment by Dunkalunk** [comment]
I wish I had more to tell you....  All the steps we've discussed have worked in the past.  If you figure it out please post it here in case others have similar issues.

Happy to still try and troubleshoot, just not sure what else to do.

**Comment by giuliano-december** [comment]
I've benn having the same issue sadly. Xbox button works, but neither vJoy or Guitar sniffer get any input from the guitar. I've got a Jaguar X1 model. 

@Dunkalunk Could you share the versions you use of WinPCap, USBPCap and vJoy please?

**Comment by Dunkalunk** [comment]
@giuliano-december absolutely and a great idea!

WinPCap - v4.1.3 - if I remember right this is the latest and last version, I don't think it's in development anymore

USBPCap - v1.5.4.0 - as far as I can tell this is still the latest

vJoy - v2.1.9.1 - appears to be the latest

I was hoping there was something to this and maybe there were updated versions causing problems but it doesn't appear to be.......

**Comment by giuliano-december** [comment]
@Dunkalunk yeah I've got the same versions :/

The guitar appears in Devices and Printers as a _HID-Compliant device_ but that's it. *Monitor vJoy* app won't respond on inputs either. Where does the guitar sniffer log the errors/info ?

I am using the binary btw. Not compiling it

**Comment by Dunkalunk** [comment]
@giuliano-december yeah it won't change how it shows up there since you connect the device to the wireless adapter and then all the packet capture reads the data into vJoy, so vJoy is really what is being used as the controller in Clone Hero.
There currently isn't any logging output that I know of.  The only time I've seen errors logged is in the console when I get it running in Jetbrains GoLand

**Comment by reedwhaley** [comment]
I'm having the same issues. Mine used to work great then I'm guessing a Windows Update happened and everything stopped working.

**Comment by sorcer0r** [comment]
Make sure you uninstall Npcap before running the application.

**Comment by tylonyong** [comment]
I followed all the steps and guitar sniffer still wasn't working.
Then I uninstalled the latest Windows update, not sure if it was 2021-03 Cumulative Update Preview for Windows 10 Version 20H2 for x64-based Systems (KB5000842). After the reboot, all the inputs in Guitar Sniffer worked again.
